sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -417,6 +417,8 @@ public abstract partial class TokenCredential
         private readonly object _dummy;
         private readonly int _dummyPrimitive;
         public TokenRequestContext(string[] scopes, string? parentRequestId = null) { throw null; }
+        public TokenRequestContext(string[] scopes, string? parentRequestId = null, string? claims = null) { throw null; }
+        public string? Claims { get { throw null; } }
```

Comment:
"Claims" is plural but not a collection while `Scopes` is. Why?

Reply:
The value of this property is actually an id token which is a complex json object containing a collection of claims which need to be included in the issued token (I will add more detail on this to the xml comments). Given that do you still have an issue with the property name being plural?

Reply:
Hm, does it follow some spec?

Reply:
https://openid.net/specs/openid-connect-core-1_0-final.html#ClaimsParameter, I can link to this directly in the doc comments.